Aliases/erlang@21

```diff
@@ -1 +1 @@
-../Formula/erlang.rb
\ No newline at end of file
+/Users/thomascioppettini/oss/homebrew-core/Formula/erlang@21.rb
```

Comment:
This alias should be removed.

Reply:
haha oops will fix

Reply:
should it be removed entirely or fixed?

Reply:
Removed. In `Aliases` we only have one alias, to the current version formula.